spec/services/persist_derivatives_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe CurationConcerns::PersistDerivatives do
+  before do
+    allow(CurationConcerns.configuration).to receive(:derivatives_path).and_return('tmp')
+  end
+
+  describe '.output_file' do
+    subject { described_class.output_file(object, destination_name, &block) }
+
+    let(:object) { double(id: '123') }
+    let(:destination_name) { 'thumbnail' }
+
+    # rubocop:disable Style/Lambda
+    let(:block) { lambda { true } }
```

Comment:
An empty stabby lambda is completely uncomprehendingly by someone who doesn't have an advanced knowledge of the language. This is an attempt to be more explicit about what we're doing.